.github/workflows/coverage-tests.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: "${{ inputs.use-next-academic-year && 'Next academic year' || 'Current academic year' }} ${{ matrix.test-type }} tests"
         run: |
-          docker compose exec -T web /bin/sh -c 'COVERAGE=true bundle exec --verbose rspec --pattern "${{ env.INCLUDE_PATTERN }}" --exclude-pattern "spec/smoke/*_spec.rb,spec/features/*_spec.rb,spec/features/**/*_spec.rb"'
+          docker compose exec -T web /bin/sh -c 'COVERAGE=true bundle exec --verbose rspec --pattern "${{ env.INCLUDE_PATTERN }}" --exclude-pattern "spec/smoke/*_spec.rb"'
```

Comment:
There's an `exclude-pattern: spec/features/*_spec.rb` line further up which is set but not used in this command. I think we need to either remove line 27 or (probably better) set it to the smoke test and change this line to use that exclude pattern.